### PR TITLE
Localise url preview

### PR DESCRIPTION
**Pull request by keunes**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -21,7 +21,8 @@ plugins:
 permalink: /:categories/:title
 
 # Localization
-languages: [en]
+languages: [en, fr]
+# don't forget to add new languages also in _data/locales.yml
```
Comment (ByteHamster) [inline _config.yml:25]: Looks unrelated

File: _includes/head.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <head>
   {% capture title %}{% t page.title %}{% endcapture %}
   {% unless title.size > 0 %}{% assign title = page.title %}{% endunless %}
-  {% capture title %}{% if title.size > 0 %}{{ title }} – {% endif %}{{ site.title }}{% endcapture %}
+  {% capture title %}{% if title.size > 0 %}{{ title }} – {{ site.title }}{% else %}{{ site.title }} – {{ t homepage.header }}{% endif %}{% endcapture %}
```
Comment (ByteHamster) [inline _includes/head.html:4]: Why this? The main page now has the title `AntennaPod -` with dash

File: _includes/head.html [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>{{ title }}</title>
-  <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
+  <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ t homepage.description }}{% endif %}">
```
Comment (ByteHamster) [inline _includes/head.html:8]: Doesn't that break the description of blog posts because it fixes it to a single, static string?

File: _data/locales.yml [hunk 4]
```diff
@@ -7,3 +7,5 @@
   name: Deutsch
 - code: nl
   name: Nederlands
+- code: fr
+  name: Français
```
Comment (ByteHamster) [inline _data/locales.yml:11]: Looks unrelated to this PR

**Comment by keunes** [comment]
@ByteHamster Sorry, should've marked it as draft. As you noted elsewhere it doesn't work indeed. Has these unrelated changes because I had to base this branch on the 'add french' branch in order to have an international page to test with. Will ping you when it's ready for review.

**Comment by keunes** [comment]
FR homepage:
afbeelding

FR contribution page:
afbeelding

FR blog post:
afbeelding

**Review by ByteHamster — APPROVED** [review]
